tests/ops/fixtures/saas/auth0_fixtures.py

```diff
@@ -39,10 +32,15 @@ def auth0_secrets():
 
 
 @pytest.fixture(scope="function")
-def auth0_identity_email():
+def auth0_identity_email(saas_config):
     return pydash.get(saas_config, "auth0.identity_email") or secrets["identity_email"]
 
 
+@pytest.fixture(scope="session")
+def auth0_erasure_identity_email():
+    return f"{cryptographic_util.generate_secure_random_string(13)}@email.com"
```

Comment:
This fixture and the `sendgrid_erasure_identity_email()` fixture in `tests/ops/fixtures/saas/sendgrid_fixtures.py` both create the same session fixture. This fixture could be moved to `conftest.py` with a generic name and shared.